README.md

```diff
@@ -1,2 +1,125 @@
 # neofs-sdk-go
-Go implementation of NeoFS SDK
+Go implementation of NeoFS SDK. It contains high-level version-independent wrappers
+for structures from [neofs-api-go](https://github.com/nspcc-dev/neofs-api-go) as well as
+helper functions for simplifying node/dApp implementations.
+
+## Repository structure
+
+### accounting
+Contains fixed-point `Decimal` type for performing balance calculations.
+
+### acl
+Contains set of constants to cover common use-cases when working with NeoFS.
+See [ACL speciifcation](https://github.com/nspcc-dev/neofs-spec/blob/master/01-arch/07-acl.md) for details.
```

Comment:
It's better not to link sprc source this way. Let's wait for a PDF published at doc site.